Sources/Base/OAuth2Securable.swift

```diff
@@ -106,6 +113,20 @@ open class OAuth2Securable: OAuth2Requestable {
 		return "http://localhost"
 	}
 	
+	/** Migrate client credentials from old token to Generic Product Name Format */
+	private func migrateToken(_ toks: OAuth2KeychainAccount, _ toks_data: [String : Any]) throws {
```

Comment:
this method name should be named as `migrateTokenAndUpdateInKeyChain` or `migrateTokenAndSaveInKeyChain`

Reply:
name is kept similar to methods used as forgetTokens, as we have separate methods for keychain for saving, removing and updating.